content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,182 @@
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
 ```
-{{< /tab >}}
 
-{{< tab "Using a Kubernetes manifest" >}}
-Create a `kind: Provider` Kubernetes manifest.  
+Install multiple services with multiple Provider objects. For example to install both AWS S3 and EC2 providers:
 
-Provide the `spec.package` location of the official provider from the Upbound Marketplace listing. 
+```yaml
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-ec2
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0
+```
+
+{{<hint "tip" >}}
+Browse the [Upbound Marketplace](https://marketplace.upbound.io/providers?tier=official) to find the provider services to install. 
+{{</ hint >}}
+
+{{< expand "Install Providers in an offline environment" >}}
 
-This example installs the [AWS official provider version 0.17.0](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.17.0).
+To install Providers in an offline environment set 
+{{<hover label="air" line="7">}}skipDependencyResolution: true{{</hover>}} in
+the Provider.
+
+```yaml {label="air"}
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+  skipDependencyResolution: true
+```
+
+When installing multiple providers, apply `skipDependencyResolution` to every
+Provider.
 
 ```yaml
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
-    name: provider-aws
+  name: upbound-provider-aws-s3
 spec:
-  package: xpkg.upbound.io/upbound/provider-aws:v0.17.0
-  packagePullSecrets:
-    - name: package-pull-secret
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+  skipDependencyResolution: true
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-ec2
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0
+  skipDependencyResolution: true
+```
+
+{{< /expand >}}
+
+View the installed providers with `kubectl get providers`
+
+```shell {copy-lines="1",label="getproviders"}
+kubectl get providers
+NAME                          INSTALLED   HEALTHY   PACKAGE                                               AGE
+upbound-provider-aws-ec2      True        True      xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0      25m
+upbound-provider-aws-s3       True        True      xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0       25m
+upbound-provider-family-aws   True        True      xpkg.upbound.io/upbound/provider-family-aws:v0.37.0   24m
+```
+
+The first provider installed of a family also installs an extra
+{{<hover label="getproviders" line="5">}}provider-family{{</hover>}} Provider.
+The `provider-family` provider manages the ProviderConfig
+for all service providers in the same family. 
```

Comment:
We should look to use the terminology "config provider" for the family's config provider package (provider-family-gcp,provider-family-azure,provider-family-aws). See https://link.excalidraw.com/l/29ZB2bl8Sjx/1LR3n45hm6p

Reply:
How about replacing 
"The `provider-family` provider manages the ProviderConfig for all service providers in the same family. "

with

This _config provider_ manages the ProviderConfig
for all service providers in the same family.

Reply:
Understood. I know they are just another provider, but do we have a preferred term to use to identify them? Specifically trying to say "a provider that is a member of a family, related to other providers of the same family and not the config provider"